src/api/type-defs/objects/ImageAttempt.ts

```diff
@@ -24,6 +24,6 @@ export default /* GraphQL */ `
     batch: String
     created: Date!
     metadata: ImageMetadata
-    errors: [ImageError!]
+    errs: [ImageError!]
```

Comment:
animl-ingest's `ingest-image` lambda [expects](https://github.com/tnc-ca-geo/animl-ingest/blob/00a0b2d6a30e0653abb284a88619aaadbcbb0faa/ingest-image/task.js#L185) there `errs` property to be there, so we should either make this a required property here and return an empty array if there are no `ImageErrors`, or update `ingest-image` to handle an undefined/null value.

Reply:
Fixed.